stdlib/LinearAlgebra/docs/src/index.md

```diff
@@ -337,6 +337,7 @@ LinearAlgebra.UnitUpperTriangular
 LinearAlgebra.UpperHessenberg
 LinearAlgebra.UniformScaling
 LinearAlgebra.I
+LinearAlgebra.(::LinearAlgebra.UniformScaling)(::Integer)
```

Comment:
```suggestion
LinearAlgebra.UniformScaling(::Integer)
```

I did a bit more digging and I think this might function as a workaround.

The issue is that internally the call syntax / functor docstring gets stored with the same key (which is just the argument signature, i.e. `Tuple{Integer}` in this case) as a normal method / constructor would. So, from Documenter's perspective, the functor syntax is not a thing and only things that exist are methods on types.

However, as there is no conflicting `UniformScaling(::Integer)` constructor, we should be able to take advantage of that and just tell Documenter to include that and it will splice in the docstring for `(::LinearAlgebra.UniformScaling)(::Integer)`.

Reply:
I am really glad you were able to find a clean solution. Thank you very much for helping!